lib/shoulda/matchers/active_record/association_matchers/required_matcher.rb

```diff
@@ -33,9 +33,9 @@ def matches?(subject)
                 end
 
               missing_option << (
-                'fail validation if ' +
```

Comment:
Is this enforced by a cop? I notice you doing this a lot and I'm wondering if it's your style or something that Rubocop is enforcing.

Reply:
This change is enforced by `1.0`, which requires string concatenation to be enforced with interpolation by default.
So to counter LineLength and [Style/StringConcatenation](https://docs.rubocop.org/rubocop/cops_style.html#stylestringconcatenation) cops, i had to use `\`.
I was thinking of `heredoc`, but that would be too much of a change, so updated it with `\` for your feedback.

Reply:
Okay, gotcha 👍